tests/testthat/test-Rules-class.R

```diff
@@ -17,28 +17,22 @@ test_that("NextBestMTD object can be created with user constructor", {
   expect_valid(result, "NextBestMTD")
 })
 
-test_that("NextBestMTD throws the error for non valid target", {
```

Comment:
what about those NextBestMTD tests? why did you remove those?

Reply:
Yes, I removed intentionally, as I realised these were a duplicated unit tests for the functionality that is provided by class validation function, and for this we already have unit tests.